openquakeplatform/vulnerability/templates/admin/vulnerability/base.html

```diff
@@ -0,0 +1,173 @@
+                          $("#search-inp").fadeOut();
+                          $("dropdown-menu").fadeOut();
+                          return false;
+                 }       
+                 stopPropagation(); 
+             } 
+             function share_click() {        
+                 if(document.getElementById('oq-share').style.display == "none") {
+                          $("#oq-share").fadeIn();
+                          $("#search-inp").fadeOut();
+                          $("dropdown-menu").fadeOut();
+                          return false;
+                  } else {
+                          $("#oq-share").fadeOut();
+                          $("dropdown-menu").fadeOut();
+                          return false;
+                  }
+             
+             }
+             function dropdown_click() {
+                 if(document.getElementById('dropdown-menu').style.display == "none") {
+                          $("#oq-share").fadeIn();
+                          $("#search-inp").fadeIn();
+                          return false; 
+                  } else {
+                          $("#oq-share").fadeOut();
+                          $("#search-inp").fadeOut();
+                          return false;
+                  }
+             }   
+      </script>    
+    </head>
+    {% load i18n %}
+
+<body class="{% if is_popup %}popup {% endif %}{% block bodyclass %}{% endblock %}">
+{% if not is_popup %}
+    {% block header %}
+    {% include "includes/header.html" %}
+    {% endblock header %}
+{% endif %}
+
+<!-- Container -->
+<div id="container">
+    {% if not is_popup %}
+{# GEM: mn: this part of the template is disabled (navigators) #}
+{% if 1 == 0 %}
+    <!-- Header -->
+    <div id="header">
+        <div id="branding">
+        {% block branding %}{% endblock %}
+        </div>
+        {% if user.is_active and user.is_staff %}
+        <div id="user-tools">
+            {% trans 'Welcome,' %}
+            <strong>{% filter force_escape %}{% firstof user.get_short_name user.get_username %}{% endfilter %}</strong>.
+            {% block userlinks %}
+                {% url 'django-admindocs-docroot' as docsroot %}
+                {% if docsroot %}
+                    <a href="{{ docsroot }}">{% trans 'Documentation' %}</a> /
+                {% endif %}
+                {% if user.has_usable_password %}
+                <a href="{% url 'admin:password_change' %}">{% trans 'Change password' %}</a> /
+                {% endif %}
+                <a href="{% url 'admin:logout' %}">{% trans 'Log out' %}</a>
+            {% endblock %}
+        </div>
+        {% endif %}
+        {% block nav-global %}{% endblock %}
+    </div>
+    <!-- END Header -->
+    {% block breadcrumbs %}
+    <div class="breadcrumbs">
+    <a href="{% url 'admin:index' %}">{% trans 'Home' %}</a>
+    {% if title %} &rsaquo; {{ title }}{% endif %}
+    </div>
+    {% endblock %}
+{% endif %}
+{# GEM: mn: end #}
```

Comment:
all the GEM block above could be removed.

Reply:
Fixed here: https://github.com/gem/oq-platform2/pull/19/commits/028d9c16b7bc4f10548dcadd5722ca32ec64b9e1